spec/core/ics-004-channel-and-packet-semantics/UPGRADES.md

```diff
@@ -499,10 +499,20 @@ function chanUpgradeConfirm(
 
 ### Cancel Upgrade Process
 
-During the upgrade handshake a chain may cancel the upgrade by writing an error receipt into the error path and restoring the original channel to `OPEN`. The counterparty must then restore its channel to `OPEN` as well. A relayer can facilitate this by sending `ChanUpgradeCancelMsg` to the handler:
+**Sub-Definitions**:
```

Comment:
I don't think it's necessary to have a subsection here for defining an aborting chain. You could just say directly something like this:
> During the upgrade handshake a chain may encounters an error and cancel the upgrade. The first channel to do so is referred to as the *aborting chain*. The aborting chain  may cancel the upgrade by writing an error receipt into the error path and restoring the original channel to `OPEN`. The counterparty must then restore its original channel to `OPEN` as well.